src/manifests/handlers/dash/segment.ts

```diff
@@ -37,26 +37,34 @@ export default async function dashSegmentHandler(
     const urlSearchParams = new URLSearchParams(event.queryStringParameters);
     const pathStem = path.basename(event.path).replace('.mp4', '');
     // Get the number part after "segment_"
-    const [, reqSegmentIndexStr, representationIdStr, bitrateStr, timeStr] =
+    // eslint-disable-next-line prefer-const
```

Comment:
I think this eslint disable can bbe removed? right? ( I saw you had `let` before )

Reply:
Very true! Removing it!